canonical-web-and-design/multipass.run
# Update multipass landing page

## Done

- Build new landing page for multipass.run (similar to MicroK8s) 
- Add JS for tabs for instructions - (Rhys asked for the Linux instructions to be visible on page load) 
- Add GitHub buttons
- Add styles for strips 
- Update Navigation

## QA

- View page at: https://multipass-run-138.demos.haus
- Check updates against [copy doc](https://docs.google.com/document/d/1uI5hsuhe5VUpFSSiQas6FVB2F7RhlbNXcr5Pvr_Lm2c/edit#)
- NOTE - Cheat sheet section not included, to be added in separate PR. Issue [here](https://github.com/canonical-web-and-design/multipass.run/issues/139)


## Issue / Card

Fixes [#3554](https://github.com/canonical-web-and-design/web-squad/issues/3554)


## Screenshots

![image](https://user-images.githubusercontent.com/58959073/105742289-39a4bb00-5f33-11eb-9fd9-6fd69c758163.png)
